lint-checks/src/main/java/com/google/maps/android/lint/checks/MapsUtilsIssueRegistry.kt

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.maps.android.lint.checks
+
+import com.android.tools.lint.client.api.IssueRegistry
+import com.android.tools.lint.detector.api.CURRENT_API
+import com.android.tools.lint.detector.api.Issue
+
+@Suppress("UnstableApiUsage")
+class MapsUtilsIssueRegistry : IssueRegistry() {
+
+    override val issues: List<Issue> =
+        listOf(GoogleMapDetector.POTENTIAL_BEHAVIOR_OVERRIDE)
+
+    override val api: Int =
+        CURRENT_API
+
+    override val minApi: Int
+        get() = 1
```

Comment:
No this API refers to the lint API, not the Android API level. Typically, this value should match `api` but that resulted in an `ObsoleteLintCustomCheck` error. Perhaps instead of going all the way back to API level 1, this should be API level 6 (one level away from the `CURRENT_API` which I have tested and verified that it works).